opennms-doc/guide-admin/src/asciidoc/text/performance-data-collection/configuration.adoc

```diff
@@ -37,3 +38,109 @@ The global behavior, especially the size of the _Thread Pool_ for _Collectd_, is
 ----
 
 <1> Size of the _Thread Pool_ to run _Collectors_ in parallel
+
+[[ga-performance-management-collectd-resource-types]]
+==== Resource Types
+.Resource Types
+
+Resource Types are used to group sets of performance data measurements for
```

Comment:
Please reformat using one sentence per line. (This makes editing and merging easier)